raiden_installer/base.py

```diff
@@ -68,6 +66,12 @@ def configuration_data(self):
         if self.routing_mode == "pfs":
             base_config.update({"pathfinding-service-address": self.path_finding_service_url})
 
+        # If the config is for a demo-env we'll need to add/overwrite some settings
+        if settings.client_release_channel == "demo_env":
+            base_config.update({"matrix-server": settings.matrix_server})
+            base_config["routing-mode"] = "pfs"
```

Comment:
this should always be the case with the Wizard, why do we need to overwrite routing-mode pfs?

Reply:
[Routing mode pfs is not always the case.](https://github.com/raiden-network/raiden-wizard/blob/master/resources/conf/mainnet.toml) It has to be set in the config file.
You could argue that this "overwrite" is not necessary. However, if you miss setting the routing-mode in your config you will a) most likely not recognize it and b) spent some time debugging (just as I did)

Reply:
no it will be always the case. The Wizard only makes sense if you want to use the pfs, right?